test/layout.ts

```diff
@@ -974,6 +976,8 @@ describe("Layout Tests", () => {
     let observer = new MutationObserver(callback);
     observer.observe(document, { childList: true, subtree: true });
 
+    config.showText = true;
```

Comment:
>  config.showText = true; [](start = 2, length = 25)

How does the test check that masking is done? Can you add a test for the case with spaces so that we mask everything now?